login-workflow/src/new-architecture/screens/ForgotPasswordScreen/ForgotPasswordScreen.tsx

```diff
@@ -0,0 +1,144 @@
+import { CheckCircle } from '@mui/icons-material';
+import { Box, SxProps, Theme, useTheme } from '@mui/material';
+import React, { useState } from 'react';
+import { Trans } from 'react-i18next';
+import { SimpleDialog } from '../../../components';
+import { useAuthContext } from '../../contexts';
+import { useLanguageLocale } from '../../hooks';
+import { SuccessScreenBase } from '../SuccessScreen';
+import { ForgotPasswordScreenBase } from './ForgotPasswordScreenBase';
+import { ForgotPasswordScreenProps } from './types';
+
+const EMAIL_REGEX = /^[A-Z0-9._%+'-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
+const emailValidator = (email: string): boolean | string =>
+    new RegExp(EMAIL_REGEX).test(email) ? true : 'Please enter a valid email';
+
+const LinkStyles = (theme: Theme): SxProps<Theme> => ({
+    fontWeight: 600,
+    color: theme.palette.primary.main,
+    textTransform: 'none',
+    textDecoration: 'none',
+    '&:visited': {
+        color: theme.palette.primary.main,
+    },
+});
+
+export const ForgotPasswordScreen: React.FC<ForgotPasswordScreenProps> = (props) => {
+    const { t } = useLanguageLocale();
+    const theme = useTheme();
+    const { actions, navigate, routeConfig } = useAuthContext();
+
+    const [emailInput, setEmailInput] = useState('');
+    const [isLoading, setIsLoading] = useState(false);
+    const [showErrorDialog, setShowErrorDialog] = useState(false);
+
+    const {
+        WorkflowCardHeaderProps: workflowCardHeaderProps = {
+            title: t('bluiAuth:HEADER.FORGOT_PASSWORD'),
+        },
+        contactPhone = '1-800-123-4567',
+        WorkflowCardInstructionProps: workflowCardInstructionProps = {
+            instructions: (
+                <Box>
+                    <Trans i18nKey={'bluiAuth:FORGOT_PASSWORD.INSTRUCTIONS_ALT'} values={{ phone: contactPhone }}>
+                        Please enter the account email associated with the account. If this email has an account with
+                        Eaton, you will receive a response within <b>one business day</b>. For urgent account issues,
+                        please call{' '}
+                        <Box component="a" href={`tel:${contactPhone}`} sx={LinkStyles(theme)}>
```

Comment:
You dont need the theme when using sx any longer. Check the LinkStyles on the login component for an example